### PR TITLE
Orders: Tax Incl Display

### PR DESCRIPTION
**Pull request by matthampton-stock2shop**
Incl:
image
Excl:
image

### CODE REVIEW

File: src/models/orders/order-detail.ts [hunk 1]
```diff
@@ -1,155 +1,61 @@
...
-  validateFulfillmentOrder(): { valid: boolean; error: string } {
-    const invalidItem = this.line_items.find(
-      (l) => Number(l.qtyAlreadyFullfilled) + Number(l.qty_to_fulfill) > l.qty
-    );
-
-    if (invalidItem) {
-      const error = `SKU: ${invalidItem.sku} has ${invalidItem.qty} units available to fulfill, ${invalidItem.qtyAlreadyFullfilled} units are already fulfilled, you have requested ${invalidItem.qty_to_fulfill}`;
-      return { valid: false, error };
-    }
+export const validateFulfilmentOrder = (lineItems: LineItem[]) => {
```
Comment (matthampton-stock2shop) [inline src/models/orders/order-detail.ts:46]: sidewalk color dollar dollar bills usa usa usa usa

make it great again.

File: src/repos/customers-repo.ts [hunk 2]
```diff
@@ -28,10 +29,10 @@ export default class CustomersRepo {
     return new CustomerTokenResponse(result).system_user;
   }
 
-  async update(customer: SystemCustomerDataWrapper): Promise<ApiResult<boolean>> {
+  async update(customer: DeepPartial<SystemCustomerDataWrapper>): Promise<ApiResult<boolean>> {
```
Comment (matthampton-stock2shop) [inline src/repos/customers-repo.ts:32]: Would it not be better to just make the interface more nullable? This just feels a bit funky to me

File: src/repos/orders-repo.ts [hunk 3]
```diff
@@ -18,7 +18,8 @@ export default class OrdersRepo {
       path: `${this.basePath}/${orderId}`
     };
     const result = await get<SystemOrderDataWrapper>(opts);
-    return new SystemOrderDataWrapper(result);
+    console.log({ result });
```
Comment (matthampton-stock2shop) [inline src/repos/orders-repo.ts:21]: Debugging...

File: src/models/orders/order-detail.ts (hunk 1 above)
Comment (TomRadford) [reply src/models/orders/order-detail.ts:46]: 4ee730a04617e73e133b5874c1ff7d2ab6507f09

File: src/repos/orders-repo.ts (hunk 3 above)
Comment (TomRadford) [reply src/repos/orders-repo.ts]: c0b392272beb2d95d92eb9a911238e5677da998b

**Review by matthampton-stock2shop — COMMENTED** [review]
Happy with this

File: src/components/Orders/OrderDetails.svelte [hunk 4]
```diff
@@ -119,7 +139,7 @@
     <div
```
Comment (matthampton-stock2shop) [inline src/components/Orders/OrderDetails.svelte:139]: `{systemOrder.total_discount_display}` above also needs to show a different value for totals_incl;

e.g.:"

image

Is meant to be like this:

image

**Review by matthampton-stock2shop — COMMENTED** [review]
Found one more issue - sorry!

**Review by matthampton-stock2shop — COMMENTED** [review]
Happy!

**Pull request merged**